docs/eventing/sources/redis/getting-started.md

```diff
@@ -0,0 +1,102 @@
+# Creating a RedisStreamSource object
+
+![version](https://img.shields.io/badge/API_Version-v1alpha1-red?style=flat-square)
+
+This topic describes how to create a `RedisStreamSource` object.
+
+## Install RedisStreamSource
+
+`RedisStreamSource` is a Knative Eventing add-on.
+
+1. Install RedisStreamSource by running the command:
+
+    ```bash
+    kubectl apply -f {{ artifact(org="knative-sandbox", repo="eventing-redis", file="redis-source.yaml") }}
+    ```
+
+1. Verify that `redis-controller-manager`is running:
+
+    ```bash
```

Comment:
Nit: such a minor thing, but I have started using `console` for these kinds of things (CLI commands, etc that are not explicitly bash). It prevents random text coloration on words that happen to be `bash` keywords.

Reply:
@abrennan89 I will create a PR for that, but fwiw, I got this from other files in this repo, so I don't think this is the first time it's been used! 😉 

```console
❯ rg "\.console"
vendor/golang.org/x/sys/unix/zsysctl_openbsd_mips64.go
14:	{"ddb.console", []_C_int{9, 6}},

vendor/golang.org/x/sys/unix/zsysctl_openbsd_386.go
14:	{"ddb.console", []_C_int{9, 6}},

vendor/golang.org/x/sys/unix/zsysctl_openbsd_amd64.go
14:	{"ddb.console", []_C_int{9, 6}},

vendor/golang.org/x/sys/unix/zsysctl_openbsd_arm64.go
14:	{"ddb.console", []_C_int{9, 6}},

vendor/golang.org/x/sys/unix/zsysctl_openbsd_arm.go
14:	{"ddb.console", []_C_int{9, 6}},

docs/getting-started/deploy-function.md
22:    ```{ .console}
28:    ```{ .console }
33:    ```{ .console .no-copy }
44:    ```{ .console}
50:    ```{ .console }
55:    ```{ .console .no-copy }

docs/functions/runtimes.md
35:```{ .console }
45:```{ .console }
52:```{ .console }

docs/functions/local-development.md
37:    ```{ .console }
43:    ```{ .console }

docs/getting-started/function-triggers.md
21:    ```{ .console}
27:    ```{ .console }
32:    ```{ .console .no-copy }
40:    ```{ .console}
47:    ```{ .console }
52:    ```{ .console .no-copy }
68:```{ .console }
73:    ```{ .console .no-copy }
87:```{ .console }
92:    ```{ .console .no-copy }
104:```{ .console }
109:    ```{ .console .no-copy }
116:```{ .console }
125:    ```{ .console .no-copy }
141:```{ .console }

docs/getting-started/creating-function.md
21:    ```{ .console}
27:    ```{ .console }
32:    ```{ .console .no-copy }
```

Reply:
@lance @lionelvillard please open an issue for docs and bring this for discussion with the docs WG either in Slack or a call before changing to this.

Everywhere else our docs use bash and it's documented in the style guide to use this, not console. I do not want to introduce inconsistencies into the docs without also updating the style guide docs and having discussed it as part of our WG.

Please consider this a blocker for merging PRs until these steps are taken, and instead use `bash` until this can be properly discussed and updated in the style guide. CC @snneji

Reply:
@lance can you please open an issue? For now I will stick to using bash until told otherwise.